src/api/reporter/MutantResult.ts

```diff
@@ -0,0 +1,13 @@
+import MutantStatus from './MutantStatus';
+import {Location} from '../core';
+
+interface MutantResult {
+  sourceFilePath: string;
+  mutationType: string;
```

Comment:
Is this the value of `Mutator.name`? If so, I think we should rename it to `mutatorName`